.env

```diff
@@ -1 +1 @@
-VITE_FIREBASE_CONFIG=
+VITE_FIREBASE_CONFIGS=
```

Comment:
this is singular in all the other apps that we use it. It would be nice to continue that convention.